mod-configuration-server/src/main/java/org/folio/rest/impl/ConfigAPI.java

```diff
@@ -53,14 +53,12 @@
 
   private final Messages            messages          = Messages.getInstance();
 
-  private String idFieldName                          = "id";
-
   public ConfigAPI(Vertx vertx, String tenantId) {
     //calculate facets on all results, this shouldnt be a performance issue as the amount of
     //records in a configuration result set shouldnt get too high
     //FacetManager.setCalculateOnFirst(0);
 
-    PostgresClient.getInstance(vertx, tenantId).setIdField(idFieldName);
+    PostgresClient.getInstance(vertx, tenantId);
   }
```

Comment:
Can you remove this constructor that has become useless? Now it only creates a PostgresClient instance and throws it away (after some idle time in the cache).

Reply:
OK